### PR TITLE
IL supports multipile device soc versions 

### PR DESCRIPTION
**Pull request by JimmyYang20**
fix issue: #229 

Signed-off-by: JimmyYang20 <yangjin39@huawei.com>

### CODE REVIEW

File: pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go [hunk 1]
```diff
@@ -700,37 +749,48 @@ func (im *Manager) getEvalResult(job *Job) ([]map[string][]float64, error) {
...
-		var deployModels []Model
-		models := im.getModelsFromJobConditions(jobConditions, sednav1.ILJobEval, sednav1.ILJobStageCondReady, "input")
-		for _, m := range models {
-			deployModels = append(deployModels, Model{Format: m.Format, URL: m.URL})
-		}
-
-		return deployModels
+		model = jobConfig.DeployModel
+	}
+	if model == nil {
```
Comment (Poorunga) [inline pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go:789]: It is redundant to write another judgment statement, put it in the default syntax block of switch：
```
switch jobStage {
    case xxx:
    ......
    default:
        return nil
}
```

Comment (JimmyYang20) [reply pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go:789]: jobStage value can not be `default`;

Comment (Poorunga) [reply pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go:789]: got it

**Pull request merged**